packages/11ty/_plugins/transforms/outputs/pdf/write.js

```diff
@@ -69,5 +72,28 @@ module.exports = (eleventyConfig) => {
     } catch (error) {
       logger.error(`Eleventy transform for PDF error compiling SASS. Error message: ${error}`)
     }
+
+    // Prevent duplicate ids by appending an incrementing digit (e.g. '_1', '_2', etc.)  to each duplicate id
+    // @TODO this should definitely be tested!
+    const elementIds = Array
+      .from(document.querySelectorAll('[id]'))
+      .map(({ id }) => id)
+
+    let duplicateIds = elementIds.reduce((duplicates, id, index, array) => {
+      if (array.indexOf(id) !== index && !duplicates.includes(id)) {
+        duplicates.push(id)
+      }
+
+      return duplicates
+    }, [])
+
+    duplicateIds.forEach((id) => {
+      const elementsWithSameId = Array.from(document.querySelectorAll(`[id=${id}]`))
+      for (const [index, element] of elementsWithSameId.entries()) {
+        if (index) {
+          element.id += `_${index}`
+        }
+      }
```

Comment:
```
      elementsWithSameId.forEach(([element, index]) => {
        if (index) element.id += `_${index}`
      })
```
though not certain why would not have an `index`, so perhaps
```suggestion
      elementsWithSameId.forEach(([element, index]) => element.id += `_${index}`)
```

Reply:
yeah this is a little funny, the `if (index)` is checking that the index is non-zero, not for the presence of an index -- this is so only subsequent duplicates get the `_#` digit appended. Maybe being more explicit with `if (index > 0)` explains intent better?
```suggestion
      elementsWithSameId.forEach((element, index) => {
        if (index > 0) element.id += `_${index}`
      })
```

Reply:
the version with the explicit `index > 0` is better, though with JSDoc would be even better still.